frontend/src/views/main/project/modals/AddTestToSuite.vue

```diff
@@ -106,6 +108,12 @@ const inputs = computed(() =>
 const mainStore = useMainStore();
 
 async function submit(close) {
+  mixpanel.track("Add test to suite", {
+    testName: test.name,
+    suiteId: anonymize(selectedSuite.value),
```

Comment:
We don't need to anonymize generated id

```suggestion
    suiteId: selectedSuite.value,
```